src/main/java/seedu/duke/TaskList.java

```diff
@@ -0,0 +1,139 @@
+package seedu.duke;
+
+import seedu.duke.task.Task;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * <h1> TaskList </h1>
+ * This TaskList class is used to store all the Task class objects in an arraylist.
+ * When Storage class loads the existing tasks from the file, the tasks are created and added to
+ * TaskList.
+ *
+ * @author Yap Xuan Xuan
+ * @version 0.1
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList() { //constructor
+        tasks = new ArrayList<>();
+    }
+
+    /**
+     * Adds the Task object to the arraylist and calls the function in Storage to store
+     * the information of the newly added task in the file. Prints out the details of the
+     * newly added task to be displayed to the user.
+     *
+     * @param task Task object to be added to the arraylist.
+     * @param isNew boolean that indicates whether the task is an existing one loaded from
+     *              the file or a newly added one.
+     */
+    public void addItem(Task task, boolean isNew) { //to append items to tasks
```

Comment:
Perhaps consider changing the method name to "addTask" for a better representation of the functions?